packages/mdc-elevation/_mixins.scss

```diff
@@ -71,11 +179,28 @@
   $penumbra-color: rgba($color, $mdc-elevation-penumbra-opacity + $opacity-boost);
   $ambient-color: rgba($color, $mdc-elevation-ambient-opacity + $opacity-boost);
 
-  @include mdc-feature-targets($feat-color) {
-    box-shadow:
-      #{"#{$umbra-z-value} #{$umbra-color}"},
-      #{"#{$penumbra-z-value} #{$penumbra-color}"},
-      #{$ambient-z-value} $ambient-color;
+  $box-shadow: (
+    #{"#{$umbra-z-value} #{$umbra-color}"},
+    #{"#{$penumbra-z-value} #{$penumbra-color}"},
+    #{$ambient-z-value} $ambient-color,
+  );
+
+  @include mdc-elevation-shadow($box-shadow, $query: $query);
+}
+
+// Private
+
+///
+/// Sets the elevation overlay transition value.
+///
+/// @param {String} $duration - The duration of the transition.
+/// @param {String} $easing - The easing function for the transition.
+/// @return {String}
+/// @access private
+///
+@mixin mdc-elevation-overlay-selector_ {
+  .mdc-elevation-overlay {
```

Comment:
Optional: would it be easier to read if we directly use `.mdc-elevation-overlay` selector?

Reply:
Could very well be. I prefer the mixin simply for the added indirection. Should the classname need to change, we'd only need to update the markup and this mixin.

I'm all ears though. If you'd prefer using the selector directly, I'll happily update it.